lisa/sut_orchestrator/azure/arm_template.bicep

```diff
@@ -98,7 +98,7 @@ func getEphemeralOSImage(node object) object => {
   name: '${node.name}-osDisk'
   diffDiskSettings: {
       option: 'local'
-      placement: 'CacheDisk'
+      placement: node.ephemeral_disk_placement_type
```

Comment:
Please check with Lili, how to apply bicep changes. It needs to compile and commit arm template changes together.